modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -456,7 +456,7 @@ function MultipleChoice(props) {
         {instructions}
         <List className={classes.optionsList}>
           {generateDefaultOptions(options, selection, disabled, isRadio, selectNonGhostOption, removeOption)}
-          <ListItem>{ghostInput}</ListItem>
+          {(input || textbox) &&  && <ListItem>{ghostInput}</ListItem>}
```

Comment:
Aside from the double `&&` typo, I'd go for something like this instead, since the conditions for displaying `ghostInput` are expanding with #589 :
```suggestion
          {ghostInput ? <ListItem>{ghostInput}</ListItem> : null}
```

Reply:
That's what I actually wrote in the beginning, but then I looked above where a similar condition is used, so I copied it from there. I think both places should be changed.

Reply:
Agreed

Reply:
Commit amended.